data/youtube.js

```diff
@@ -41,18 +40,14 @@ export async function fetchChannelSection() {
   const { data = {} } =
     (await axios.get(`${url}/channelSections`, {
       params: {
-        channelId: CHANNEL_ID,
+        id: CHANNEL_SECTION_ID,
         key: API_KEY,
-        part: 'id,snippet,contentDetails',
+        part: 'contentDetails',
       },
     })) || {};
 
-  const [channelSection] = data.items.filter(
-    ({ snippet: { title } = {} } = {}) => title === TARGET_SECTION_TITLE
-  );
-
   const channelSectionPlaylistIDs =
-    channelSection?.contentDetails?.playlists || [];
+    data?.items[0]?.contentDetails?.playlists || [];
```

Comment:
💁 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining#Array_item_access_with_optional_chaining
```suggestion
    data?.items?.[0]?.contentDetails?.playlists || [];
```